.github/drafts/16298/(Rehaul)-Docker-Outline.md

```diff
@@ -0,0 +1,65 @@
+
+## Introduction
+ * Docker Intro
+ * Why Use Docker 
+
+## Prerequisites 
+The versions and files required to install Docker will be covered in this section.
+//cross-link to docker docs
+* Docker
+* Docker-Compose 
+
+### Steps to check the installed versions
+
+## Get started with Docker compose (H1)
+In this section, we will discuss how to configure docker-compose with a .yml file and with the curl command. 
+### Introduction 
+* Understanding docker-compose files
+
+### Docker-compose configuration (H2)
+* .yml file setup
+
+### Start your containers (H2)
+We will run the containers in this section and split it into two sections (CE and BE)
+* Community Edition (H3)
+  * Configuration
+
+* Business Edition (H3)
+  * Configuration
+
+### Setting up the app/Getting Started  (H2)
+What users can do after container installation will be covered in this section. (Example: explanations on how to set up the first app, details on the admin settings page, etc.)
+* Set up (with video example)
+* Admin setting page info
+* Deploy
+
+### Updating Appsmith (H2)
+
+## Enabling Appsmith Auto-updates (H1)
+
+### Explore Appsmith (without docker-compose)
+Here we will explain how user can quickly get Appsmith up and running
+* Introduction
+
+* Community Edition (H2)
+  * Installation 
+  * Updating Appsmith (without docker-compose)
+  * Restarting Containers
+
+* Business Edition 
+   * Installation
+   * Updating Appsmith (without docker-compose)
+  * Restarting Containers
+
+//cross link -Upgrade to Business Edition
+
```

Comment:
Here, the Updates and Auto Updates section can be placed with info on how to do it with Docker Compose and Docker run